addons/binding/org.openhab.binding.freebox/src/main/java/org/openhab/binding/freebox/handler/FreeboxThingHandler.java

```diff
@@ -61,23 +65,38 @@
     private FreeboxHandler bridgeHandler;
     private Calendar lastPhoneCheck;
     private String netAddress;
+    private String airPlayName;
+    private String airPlayPassword;
 
     public FreeboxThingHandler(Thing thing) {
         super(thing);
-
-        phoneJob = null;
-        callsJob = null;
-        bridgeHandler = null;
-        netAddress = null;
     }
 
     @Override
     public void handleCommand(ChannelUID channelUID, Command command) {
+        if (getThing().getStatus() != ThingStatus.ONLINE) {
```

Comment:
You should never do such a check!
Instead, you should try to reach your device and only decide afterwards whether you have to keep the status OFFLINE (because you still cannot reach it) or if you can switch it to ONLINE again.